examples/calculations/Sounding_Calculations.py

```diff
@@ -0,0 +1,210 @@
+            ptop = p[i]
+            htop = h[i]
+            break
+
+    if height_layer:
+        return hbot, htop
+    else:
+        return pbot, ptop
+
+
+###########################################
+# Upper air data can be obtained using the siphon package, but for this example we will use
+# some of MetPy's sample data.
+col_names = ['pressure', 'height', 'temperature', 'dewpoint', 'direction', 'speed']
+
+df = pd.read_fwf(get_test_data('20110522_OUN_12Z.txt', as_file_obj=False),
+                 skiprows=7, usecols=[0, 1, 2, 3, 6, 7], names=col_names)
+
+# Drop any rows with all NaN values for T, Td, winds
+df = df.dropna(subset=('temperature', 'dewpoint', 'direction', 'speed'
+                       ), how='all').reset_index(drop=True)
+
+###########################################
+# Isolate needed variables from our data file and attach units
+p = df['pressure'].values * units.hPa
+T = df['temperature'].values * units.degC
+Td = df['dewpoint'].values * units.degC
+wdir = df['direction'].values * units.degree
+sped = df['speed'].values * units.knot
+height = df['height'].values * units.meter
+
+###########################################
+# Compute the wind components
+u, v = mpcalc.wind_components(sped, wdir)
+
+###########################################
+# Compute common sounding index parameters
+ctotals = mpcalc.cross_totals(p, T, Td)
+kindex = mpcalc.k_index(p, T, Td)
+showalter = mpcalc.showalter_index(p, T, Td)
+total_totals = mpcalc.total_totals_index(p, T, Td)
+vert_totals = mpcalc.vertical_totals(p, T)
+
+###########################################
+# Compture the parcel profile for a surface-based parcel
+prof = mpcalc.parcel_profile(p, T[0], Td[0])
+
+###########################################
+# Compute the corresponding LI, CAPE, CIN values for a surface parcel
+lift_index = mpcalc.lifted_index(p, T, prof)
+cape, cin = mpcalc.cape_cin(p, T, Td, prof)
+
+###########################################
+# Determine the LCL, LFC, and EL for our surface parcel
+lclp, lclt = mpcalc.lcl(p[0], T[0], Td[0])
+lfcp, _ = mpcalc.lfc(p, T, Td)
+el_pressure, _ = mpcalc.el(p, T, Td, prof)
+
+###########################################
+# Compute the characteristics of a mean layer parcel (50-hPa depth)
+ml_t, ml_td = mpcalc.mixed_layer(p, T, Td, depth=50 * units.hPa)
+ml_p, _, _ = mpcalc.mixed_parcel(p, T, Td, depth=50 * units.hPa)
+mlcape, mlcin = mpcalc.mixed_layer_cape_cin(p, T, prof, depth=50 * units.hPa)
+
+###########################################
+# Compute the characteristics of the most unstable parcel (50-hPa depth)
+mu_p, mu_t, mu_td, _ = mpcalc.most_unstable_parcel(p, T, Td, depth=50 * units.hPa)
+mucape, mucin = mpcalc.most_unstable_cape_cin(p, T, Td, depth=50 * units.hPa)
+
+###########################################
+# Compute the Bunkers Storm Motion vector and use to calculate the critical angle
+bunkers = mpcalc.bunkers_storm_motion(p, u, v, height)
+u_storm, v_storm = bunkers[0]
```

Comment:
```suggestion
(u_storm, v_storm), *_ = mpcalc.bunkers_storm_motion(p, u, v, height)
```

Reply:
Nice! I don't think I've seen unpacking done in that way. Elegant.